tests/modules/configurations/resources/utils.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""
+Configuration resources utils
+-------------
+"""
+from tests import utils as test_utils
+
+EXPECTED_KEYS = {'response', 'success'}
+CONFIG_PATH = '/api/v1/configuration/default'
+CONFIG_DEF_PATH = '/api/v1/configurationDefinition/default'
+
+
+def read_configuration(
+    flask_app_client,
+    user,
+    conf_key,
+    expected_status_code=200,
+):
+    return test_utils.get_dict_via_flask(
+        flask_app_client,
+        user,
+        scopes='configuration:read',
+        path=f'{CONFIG_PATH}/{conf_key}',
+        expected_status_code=expected_status_code,
+        response_200=EXPECTED_KEYS,
+        response_error={'success', 'message'},
+    )
```

Comment:
Why don't you validate success is correct in these functions too?
You know that there must be a success and you know what the expected status code is. 
https://github.com/WildMeOrg/houston/pull/329/files#diff-166543162d1504dd37615789b714eb76c5b8bcda8f575610465d690f84c7925aR63-R69
Then you can remove the success validation from the calling code too.